lib/cc/engine/csslint.rb

```diff
@@ -48,14 +48,31 @@ def results
         @results ||= Nokogiri::XML(csslint_xml)
       end
 
+      def build_files_with_exclusions(exclusions)
+        files = Dir.glob("**/*.css")
+        files.reject { |f| exclusions.include?(f) }
+      end
+
+      def build_files_with_inclusions(inclusions)
+        inclusions.map do |include_path|
+          if include_path =~ %r{/$}
```

Comment:
It would be, but we're not using activesupport here and keeping things light might be a higher priority here.